api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -219,19 +199,16 @@
             .sortOrder(SortOrder.ASC)
             .sortColumn(FilterColumns.PARTICIPANTID.toString());
 
-    List<DbParticipantCohortStatus> paginatedPCS =
+    List<ParticipantCohortStatus> paginatedPCS =
         cohortReviewService.findAll(cohortReview.getCohortReviewId(), pageRequest);
 
-    org.pmiops.workbench.model.CohortReview responseReview =
-        cohortReviewMapper.dbModelToClient(cohortReview, pageRequest);
-    responseReview.setParticipantCohortStatuses(
-        paginatedPCS.stream()
-            .map(
-                pcs ->
-                    participantCohortStatusMapper.dbModelToClient(
-                        pcs, cohortBuilderService.findAllDemographicsMap()))
-            .collect(Collectors.toList()));
-    return ResponseEntity.ok(responseReview);
+    cohortReview
+        .page(pageRequest.getPage())
+        .pageSize(pageRequest.getPageSize())
+        .sortOrder(pageRequest.getSortOrder().toString())
+        .sortColumn(pageRequest.getSortColumn())
+        .participantCohortStatuses(paginatedPCS);
```

Comment:
I will create a future story that will remove page, pageSize, sortOrder and sortColumn for the CohortReview object